src/fragments/lib/auth/flutter/signin_with_custom_flow/10_cli_setup.mdx

```diff
@@ -0,0 +1,182 @@
+
+### Setting-up custom auth flow with the Amplify CLI
+
+The Amplify CLI can help you setup the AWS Lambda triggers for you custom authentication flow. In the terminal, navigate to your project, run `amplify add auth`, and choose the following options:
+
+```terminal
+? Do you want to use the default authentication and security configuration? Manual configuration?
+    `Select the authentication/authorization services that you want to use: User Sign-Up, Sign-In, connected with AWS IAM controls (Enables per-user Storage features for images or other content, Analytics, and more)`
+? Please provide a friendly name for your resource that will be used to label this category in the project:
+    `<hit enter to take default or enter a custom label>`
+? Please enter a name for your identity pool.
+    `<hit enter to take default or enter a custom name>`
+? Allow unauthenticated logins? (Provides scoped down permissions that you can control via AWS IAM)
+    `No`
+? Do you want to enable 3rd party authentication providers in your identity pool?
+    `No`
+? Please provide a name for your user pool:
+    `<hit enter to take default or enter a custom name>`
+? How do you want users to be able to sign in?
+    `Username`
+? Do you want to add User Pool Groups?
+    `No`
+? Do you want to add an admin queries API?
+    `No`
+? Multifactor authentication (MFA) user login options:
+    `OFF`
+? Email based user registration/forgot password:
+    `Enabled (Requires per-user email entry at registration)`
+? Please specify an email verification subject:
+    `Your verification code`
+? Please specify an email verification message:
+    `Your verification code is {####}`
+? Do you want to override the default password policy for this User Pool?
+    `No`
+? What attributes are required for signing up?
+    `Email`
+? Specify the app's refresh token expiration period (in days):
+    `30`
+? Do you want to specify the user attributes this app can read and write?
+    `No`
+? Do you want to enable any of the following capabilities?
+    `NA`
+? Do you want to use an OAuth flow?
+    `No`
+? Do you want to configure Lambda Triggers for Cognito?
+    `Yes`
+? Which triggers do you want to enable for Cognito?
+    `Create Auth Challenge, Define Auth Challenge, Verify Auth Challenge Response`
+? What functionality do you want to use for Create Auth Challenge?
+    `Custom Auth Challenge Scaffolding (Creation)`
+? What functionality do you want to use for Define Auth Challenge?
+    `Custom Auth Challenge Scaffolding (Definition)`
+? What functionality do you want to use for Verify Auth Challenge Response?
+    `Custom Auth Challenge Scaffolding (Verification)`
+
+? Do you want to edit your boilerplate-create-challenge function now?
+    `Yes`
+? Please edit the file in your editor: <local file path>/src/boilerplate-create-challenge.js
+```
+
+The boilerplate for Create Auth Challenge opens in your selected code editor. Enter the following code to this file:
```

Comment:
Can we be clearer about the location of the file in relation to the project folder here?

Reply:
The CLI offers to open the file for you. We don't go into details about the file location in our other docs.